.gitignore

```diff
@@ -19,3 +19,5 @@ static/packs/*
 dist/
 
 docs/
+src/module/item/.DS_Store
+*/.DS_Store
```

Comment:
Add this to your personal .git/info/exclude rather than the repository .gitignore

Reply:
Fixed; thanks.